tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -587,10 +588,6 @@ private void validateRequest(DAGRequest dagRequest) {
     // check encode type
     requireNonNull(dagRequest.getEncodeType());
 
-    // A DAG request must contain a valid timestamp
-    if (dagRequest.getStartTs() == 0) {
```

Comment:
why is this line deleted?

Reply:
because we do not need validate start ts any more?

Reply:
okay, so where do we get the correct start ts in means of different version of TiKV?

Reply:
We set startTs in setStartsFallback if we meet an older version of TiKV.

Reply:
We set startTs in setStartsFallback if we meet an older version of TiKV.